.github/workflows/reusable-e2e.yml

```
@@ -74,6 +78,17 @@ jobs:
           docker logs -f "${container_id}" &
           exit $(docker wait "${container_id}")
 
+      - name: Start Test (second run)
+        if: inputs.run-twice
+        run: make ${{ inputs.make-target }}
+
+      - name: Watch Test (second run)
+        if: inputs.run-twice
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:3:6: Quote this to prevent word splitting

(shellcheck)